images/1try/Scontainer.css

```diff
@@ -0,0 +1,40 @@
+*{
+    box-sizing: border-box;
+    scroll-behavior: smooth;
+}
+.container{
+    display: grid;
+    grid-template:
+    "leftmenu   mainMenu rightmenu" 500px/
+     300px  50px 400px 300px;
+    border:black solid 2px;
+}
+.icons{
+    display: grid;
+    grid-template-rows: repeat(5 ,auto);
+
+}
+body{
+    background:chocolate;
```

Comment:
I think it's `background-colour: chocolate`. I have never used solely the word `background` before. I might try your way later on.